### DISCUSSION TITLE
No Over Land Campaign for Marianas?

### DISCUSSION THREAD

**Discussion by GrEaSeLiTeNiN** [Khopa/dcs_liberation#1937] [Q&A] [answered]
If I recall right, all 3 campaigns in Marianas appear to be out at sea. It's a nice map and I would like to fly missions over the terrain and coastal areas. Am I missing something? Thks

**Comment by Starfire13** [comment, marked as answer]
There are none available as pathfinding is an issue. By the way, I suggest getting on the Liberation Discord. It's a lot easier responding to questions on there.

**Reply by MetalStormGhost** [reply]
The Mount Barrigada campaign can be flown with fully land-based factions.
